components/external-link/index.js

```diff
@@ -1,15 +1,14 @@
 import React from "react"
 import PropTypes from "prop-types"
 import clsx from "clsx"
-import styles from "./styles.module.scss"
 
 /**
  * Creates an anchor tag <a> that opens the given URL in a new window/tab
  */
 export const ExternalLink = ({ url, className, children }) => (
   <a
     href={ url }
-    className={ clsx([styles.externalLink, className])}
+    className={ clsx([className])}
```

Comment:
I can't believe you missed this empty space!

Reply:
I can't either ... eslint should've caught that and told me about it. I'll have to look into that configuration to make sure it catches that kinda stuff in the future. Thanks for letting me know about it. Will be pushing up a fix for it.